File: CS/DXApplication5/Program.cs
using DevExpress.UserSkins;
using System;
using System.Linq;
using System.Windows.Forms;

namespace DXSample {
    static class Program {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main() {
            BonusSkins.Register();
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new Main());
        }
    }
}

File: VB/DXApplication5/Program.vb
Imports DevExpress.UserSkins
Imports System
Imports System.Linq
Imports System.Windows.Forms

Namespace DXSample
	Friend Module Program
		''' <summary>
		''' The main entry point for the application.
		''' </summary>
		<STAThread>
		Sub Main()
			BonusSkins.Register()
			Application.EnableVisualStyles()
			Application.SetCompatibleTextRenderingDefault(False)
			Application.Run(New Main())
		End Sub
	End Module
End Namespace